kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/preprocess/SinkRecordFilterTest.java

```diff
@@ -0,0 +1,198 @@
+package com.wepay.kafka.connect.bigquery.preprocess;
+
+import com.jayway.jsonpath.InvalidPathException;
+import com.wepay.kafka.connect.bigquery.SinkConnectorPropertiesFactory;
+import com.wepay.kafka.connect.bigquery.config.BigQuerySinkConfig;
+import org.apache.log4j.AppenderSkeleton;
+import org.apache.log4j.Logger;
+import org.apache.log4j.spi.LoggingEvent;
+import org.hamcrest.CoreMatchers;
+import org.hamcrest.Matcher;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+public class SinkRecordFilterTest {
+    private static final String INCLUDE_CONDITION_CONFIG = "includeCondition";
+    private static final String EXCLUDE_CONDITION_CONFIG = "excludeCondition";
+    private static Map testMap;
+    private SinkRecordFilter sinkRecordFilter;
```

Comment:
since you explicitly initialize the filter in each test, it probably makes more sense for this to be a local variable in each test.